packages/open-next/src/build/edge/createEdgeBundle.ts

```
@@ -82,9 +82,9 @@ export async function buildEdgeBundle({
             converter: override("converter") ?? defaultConverter,
             ...(includeCache
               ? {
-                  tagCache: override("tagCache") ?? "dynamodb-lite",
-                  incrementalCache: override("incrementalCache") ?? "s3-lite",
-                  queue: override("queue") ?? "sqs-lite",
+                  tagCache: override("tagCache") ?? "dummy",
```

This is not what we want to do at all. 
We should fallback to dummy **only** when tagCache is a function (same for the others). When nothing is provided, we should still use the current one

---

Yeah, and sorry about the confusion. Of course we need to provide default values if your building from AWS